pact/ism/merkle-tree-ism.pact

```diff
@@ -22,26 +22,19 @@
   (defcap ONLY_ADMIN () (enforce-guard "NAMESPACE.bridge-admin"))
 
   (defun initialize:string (validators:[string] threshold:integer)
-    (with-capability (ONLY_ADMIN)
-      (if (and 
-            (= 
-              (length validators) 
-              (length (distinct validators))
-            )
-            (> threshold 0) 
-          )
-          (insert contract-state "default"
-            {
-                "validators": validators,
-                "threshold": threshold
-            }
-          )
-          "Invalid validators or threshold"
-      )
-    )
-  )
+  (enforce (> (length validators) 0) "Empty list of validators is not allowed")

```

Comment:
This one seems to be important

Reply:
It was correct by construction, but yeah, it'd be good to have this failsafe. +1 from me